app/assets/javascripts/discourse/app/templates/components/sidebar/categories-section.hbs

```diff
@@ -3,9 +3,8 @@
   @headerRoute="discovery.categories"
   @headerLinkText={{i18n "sidebar.sections.categories.header_link_text"}}
   @headerLinkTitle={{i18n "sidebar.sections.categories.header_link_title"}}
-  @headerAction={{this.editTracked}}
-  @headerActionTitle={{i18n "sidebar.sections.categories.header_action_title"}}
-  @headerActionIcon="pencil-alt" >
+  @headerActions={{array (hash action=this.editTracked title=(i18n "sidebar.sections.categories.header_action_title"))}}
```

Comment:
I think we should be able to pass a non-array here and just let the API convert it into an array.